src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyPrice = 40;
+  const lowDiscount = 20;
+  const bigDiscount = 50;
+
+  const result = days * dailyPrice;
+
+  if (days < 3) {
+    return result;
+  }
+
+  if (days < 7) {
+    return result - lowDiscount;
+  }
+
+  if (days >= 7) {
+    return result - bigDiscount;
+  }
+
+  return result;
```

Comment:
u really need 4 returns? Mb, we can decrease their count?)

Reply:
Andreeeeeeeew!!!!

Reply:
oops😂